library_management/main.py

```diff
@@ -0,0 +1,61 @@
+from typing import List
+
+import uvicorn
+from fastapi import FastAPI, Depends, HTTPException, Query
+from sqlalchemy.orm import Session
+from database import engine, SessionLocal, Base
+from crud import create_author, get_authors, get_author, create_book, get_books, get_books_by_author
+from schemas import AuthorCreate, Author, AuthorList, BookCreate, Book
+
+app = FastAPI()
+
+
+Base.metadata.create_all(bind=engine)
+
+
+def get_db():
+    db = SessionLocal()
+    try:
+        yield db
+    finally:
+        db.close()
+
+
+@app.post("/authors/", response_model=Author)
+def create_new_author(author: AuthorCreate, db: Session = Depends(get_db)):
+    return create_author(db, author)
+
+
+@app.get("/authors/", response_model=AuthorList)
+def list_authors(skip: int = Query(0), limit: int = Query(10), db: Session = Depends(get_db)):
+    authors = get_authors(db, skip, limit)
+    return {"items": authors}
+
+
+@app.get("/authors/{author_id}", response_model=Author)
```

Comment:
```suggestion
@app.get("/authors/{author_id}/", response_model=Author)
```

the same in other similar cases